test/auth_section/create_account/widget_test.dart

```diff
@@ -14,23 +13,23 @@ class MockNavService extends Mock implements NavigationService {}
 
 void main() {
   Widget makeTestableCreateAccountWidget(
-      NavigationService mockNav, MockFirebaseAuthService mockAuth) {
+      MockNavService mockNavService, FakeFirebaseAuthService mockAuth) {
```

Comment:
So in my mind `FakeFirebaseAuthService mockAuth` should be `FakeAuthService fakeAuth`, but: 
- please just ignore me if you prefer a different naming pattern 
- am I right in thinking we have an abstract AuthService and a FirebaseAuthService that implements the interface? I can't remember... if so, same comment as earlier - the fake doesn't need to know how the service is implemented  so it makes more sense to fake the AuthService IMO

Reply:
Actually given that you did `MockNavService mockNavService` I guess the other fake would be `FakeAuthService fakeAuthService`

Reply:
I get a bit hung up on naming, as I'm sure you can tell - I just think it's super important given how much time we spend trying to understand existing code. Please don't hesitate to ignore my suggestions

Reply:
Yes you're right! Idk why I chose to mock the firebaseauthservice over the abstract class.